### ISSUE TITLE
Feature request - icons on the popup 

### ISSUE THREAD

**Issue by Ykrm** [traccar/traccar-web#1117]
Hello,

I suggest to add icons to each item on the popup to have a better visual and good ergonomics.
https://ibb.co/FbVt3f6

**Comment by tananaev** [comment]
There is probably a 100 different items available there, so it's going to be hard to find an icon for each.

**Comment by Ykrm** [comment]
Yes i agree but it's doable at least for main items displayed on the popup (not details) like speed motion battery ignition distance

**Comment by tananaev** [comment]
I think you misunderstood. The content of the popup is configurable.

**Comment by sandreli** [comment]
I dont want to open another thread but I think the popup can be improved in a way without the need to find an icon for each.

It would be very nice if the system would permit adding images/svg/icons to the attributes and computed attributes.
I think this would improve a lot on the customization possibilities and give the option to make a more visually appealing webui/app, while at the same time keeping the default product clean.

example 1: instead of computing attributes for Signal/RSSI with poor, good,normal...we could add links to different colored icons. in a way, similar to the course which is coded.

example 2: adding an icon to the beginning of the attribute name in order to make the pop-up info easier to read at a quick glance.

If security is a concern, the links/code can be added in the xml config and just give the posibility to use the variables in the configuration.

Also, the posibility to rearrange that popup in different ways and even add columns would improve a lot (issue #4879)

I am saying this because usually the clients will use the popup information we configure and not the "more details" where all parameters are shown like all io's and such.

**Comment by tananaev** [comment]
You can already rearrange the data in the popup, if you didn't know.

**Comment by tananaev** [comment]
You are mistaken. You can reposition it.

You cannot add columns or images. That is correct and I'm not arguing that.

**Comment by sandreli** [comment]
You are right. I was thinking about the initial position that there is no server configuration where to initially display the popup (below the device or exact screen coordinates...). As once you get out of the menu it looses the position. But was not clear.

**Comment by tananaev** [comment]
That is a completely different story and I wouldn't call it "re-arrange".
